paasta_tools/cli/cmds/status.py

```diff
@@ -2241,6 +2253,20 @@ def status_marathon_job_human(
         )
 
 
+def _get_paasta_status_version(args, system_paasta_config) -> bool:
```

Comment:
Since this returns a boolean, can we call it something that explains what the boolean is like "_use_new_paasta_status()"?